src/pytest_qgis/pytest_qgis.py

```diff
@@ -10,14 +13,29 @@
 from pytest_qgis.mock_qgis_classes import MainWindow, MockMessageBar
 from pytest_qgis.qgis_interface import QgisInterface
 
+if TYPE_CHECKING:
+    from pathlib import Path
```

Comment:
Do you suggest importing all the imports needed only in typing here?

Reply:
If so, should `QgisInterfaceOrig` be added here as well?

Reply:
Yes, that's a good convention.
Force pushed a new commit.